### ISSUE TITLE
Change initial interaction hint (i.e., grab hint) in PDOM

### ISSUE THREAD

**Issue by terracoda** [phetsims/friction#150]
As discussed in today's meeting, please shorten the initial grab hint from:
-  “Grab Chemistry book, then rub books together fast or slow, and move down to rub harder.” 
to 
- "Grab Chemistry book to play."

The above hint still changes to "Reset the sim to make more observations." as before.

The grab alerts will take care of the specific moving and rubbing cues.

**Comment by terracoda** [comment]
Changed above hint to be "Grab Chemistry book to play."
Apologies for missing "Chemistry".

**Comment by zepumph** [comment]
Implemented, @terracoda  please review.

**Comment by terracoda** [comment]
This sounds great @zepumph. Thanks.
Closing.
